common/src/enums/sponsorshipStatus.ts

```diff
@@ -0,0 +1,6 @@
+export enum SponsorshipStatus {
+  Requested = 0, // self-hosted only
+  Sent = 1,
+  Active = 2,
+  ToRevoke = 3,
```

Comment:
I left in `Requested` here because we don't include a Sync Date to the client that we could use to communicate that it is on self hosted and hasn't synced yet like the figma shows.